simulations/simulation_runner.py

```diff
@@ -30,14 +27,16 @@ def __init__(
         else:
             self.report_interval = 1
 
-        self.network = Network(validator_set, protocol, force_justify_messages)
+        self.network = network
 
         self.plot_tool = protocol.PlotTool(display, save, self.network.global_view, validator_set)
         self.plot_tool.plot()
 
     def run(self):
         """ run simulation total_rounds if specified
             otherwise, run indefinitely """
+        self._send_initial_messages()
```

Comment:
Requirement for the integer and binary protocols. Messages are only sent to the network when generated, but the integer validators come loaded up with their initial bet. Without attempting to send these initial messages, we never have enough info to add any later messages to justified_messages.

This workaround forces all the blockchain validators to try to send the genesis block. This ends up being fine because when validators receive it, they just don't care because they already have the genesis